ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/l2_protocol_forwarding.schema.yml

```diff
@@ -0,0 +1,91 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  l2_protocol:
+    documentation_options:
+      filename: data_model/L2 Protocol Forwarding
+    type: dict
+    keys:
+      forwarding_profiles:
+        type: list
+        primary_key: name
+        items:
+          type: dict
```

Comment:
Could this be a list of protocol with the protocol name as primary_key and valid_values for existing protocol rather than having it as a dict:

cf EOS output:

```
Leaf1(config-l2-protocol)#forwarding profile TEST
Leaf1(config-l2p-fwd-profile-TEST)#show cli commands
APPEND [ COMMENT ]
[no|default] PROTOCOL [ TAGFORMAT ] ACTION
[no|default] comment
do CMD
exit
show active
show active all [ detail ]
show comment
```

Reply:
Agree, so
```yaml
l2_protocol:
  forwarding_profiles:
    - name: TEST1
      protocols: 
        - name: bfd_per_link_rfc_7130 <-- This field should be primary_key and then also add valid_values for the protocols we know.
          forward: bool
          tagged_forward: bool
          untagged_forward: bool
```

Reply:
Made the change to the data model, also tried to improve the doc template.